gapic-common/lib/gapic/common/version.rb

```diff
@@ -14,6 +14,6 @@
 
 module Gapic
   module Common
-    VERSION = "0.4.3".freeze
+    VERSION = "0.4.4".freeze
```

Comment:
In fact, we should probably bump the minor version to 0.5.0 because this is a new feature that the usage will depend on. (The generated libraries will need to depend on `~> 0.5`)

Reply:
updated